lib/chef-dk/helpers.rb

```diff
@@ -74,6 +73,22 @@ def omnibus_chefdk_location
       @omnibus_chefdk_location ||= File.expand_path('embedded/apps/chef-dk', expected_omnibus_root)
     end
 
+    def chefdk_home
+      @chefdk_home ||= begin
+                         chefdk_home_set = !([nil, ''].include? ENV['CHEFDK_HOME'])
+                         if chefdk_home_set
+                           ENV['CHEFDK_HOME']
+                         else
+                           old_home = File.join(Gem.user_home, '.chefdk')
```

Comment:
I don't think backcompat is strictly require if this only affects Policyfiles. We can just document it in the release announcement.